sync/sync.go

```diff
@@ -54,16 +58,39 @@ func New(bc *blockchain.Blockchain, starkNetData starknetdata.StarknetData,
 		log:                 log,
 		pendingPollInterval: pendingPollInterval,
 
-		opTimers: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		opTimerHistogram: prometheus.NewHistogramVec(prometheus.HistogramOpts{
 			Namespace: "sync",
 			Name:      "timers",
 		}, []string{"op"}),
-		totalBlocks: prometheus.NewCounter(prometheus.CounterOpts{
+		blockCount: prometheus.NewCounter(prometheus.CounterOpts{
 			Namespace: "sync",
 			Name:      "blocks",
 		}),
+		chainHeightGauge: prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace: "sync",
+			Name:      "blockchain_height",
+		}),
+		bestBlockGauge: prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace: "sync",
+			Name:      "best_known_block_number",
+		}),
+		reorgCount: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: "sync",
+			Name:      "reorganisations",
```

Comment:
Although I did change it to satisfy uk based linter, this was based on [nethermind_reorganizations](https://docs.nethermind.io/nethermind/ethereum-client/metrics/blockchain)

Reply:
That's ok. We use UK spelling elsewhere.